crates/pixi_command_dispatcher/src/instantiate_tool_env/mod.rs

```diff
@@ -230,6 +230,17 @@ impl InstantiateToolEnvironmentSpec {
             .await
             .map_err_with(Box::new)
             .map_err_with(InstantiateToolEnvironmentError::SolveEnvironment)?;
+
+        let packages_str = solved_environment
+            .iter()
+            .map(|p| {
+                let record = p.package_record();
+                format!("{}={}", record.name.as_normalized(), record.version)
+            })
+            .collect::<Vec<_>>()
+            .join(", ");
+        tracing::info!("Packages in tool environment: {}", packages_str);
```

Comment:
```suggestion
    if tracing::event_enabled!(Level::INFO) {
        let packages_str = solved_environment
            .iter()
            .map(|p| {
                let record = p.package_record();
                format!("{}={}", record.name.as_normalized(), record.version)
            })
            .collect::<Vec<_>>()
            .join(", ");
        info!("Packages in tool environment: {}", packages_str);
}
```

Maybe something like this? To avoid doing work that isn't actually used